parser_tool/api.py

```diff
@@ -266,4 +265,59 @@ def post(self, request):
 
         logger.debug("lemmatize response data=%s" % result)
 
+        return JsonResponse(result, safe=False)
+
+
+class getSimilarLSH(BaseAPIView):
```

Comment:
One minor nitpick and a suggestion:
- nit: The class name should ideally be uppercased (by convention), so `GetSimilarLSH`.
- suggestion: We should think about adding docstrings to new view classes with a brief description (one or two sentences), just so it's clear what tool is expected to use this API endpoint and what kind of data it's returning. This will be helpful as the tools expand.